exec/java-exec/src/main/java/org/apache/drill/exec/server/rest/profile/FragmentWrapper.java

```diff
@@ -136,25 +215,25 @@ public String getContent() {
         biggestBatches = Math.max(biggestBatches, batches);
       }
 
-      builder.appendCell(new OperatorPathBuilder().setMajor(major).setMinor(minor).build(), null);
-      builder.appendCell(minor.getEndpoint().getAddress(), null);
-      builder.appendMillis(minor.getStartTime() - start, null);
-      builder.appendMillis(minor.getEndTime() - start, null);
-      builder.appendMillis(minor.getEndTime() - minor.getStartTime(), null);
+      builder.appendCell(new OperatorPathBuilder().setMajor(major).setMinor(minor).build(), null, null);
+      builder.appendCell(minor.getEndpoint().getAddress(), null, null);
+      builder.appendMillis(minor.getStartTime() - start, null, null);
+      builder.appendMillis(minor.getEndTime() - start, null, null);
+      builder.appendMillis(minor.getEndTime() - minor.getStartTime(), null, null);
 
-      builder.appendFormattedInteger(biggestIncomingRecords, null);
-      builder.appendFormattedInteger(biggestBatches, null);
+      builder.appendFormattedInteger(biggestIncomingRecords, null, null);
+      builder.appendFormattedInteger(biggestBatches, null, null);
 
-      builder.appendTime(minor.getLastUpdate(), null);
-      builder.appendTime(minor.getLastProgress(), null);
+      builder.appendTime(minor.getLastUpdate(), null, null);
+      builder.appendTime(minor.getLastProgress(), null, null);
 
-      builder.appendBytes(minor.getMaxMemoryUsed(), null);
-      builder.appendCell(minor.getState().name(), null);
+      builder.appendBytes(minor.getMaxMemoryUsed(), null, null);
+      builder.appendCell(minor.getState().name(), null, null);
```

Comment:
+1 for Succinct way to convey what the function call is. I'll incorporate additional methods.